src/Util/ClassSourceManipulator.php

```diff
@@ -1137,22 +1137,22 @@ private function addNodesToSetOtherSideOfOneToOne(RelationOneToOne $relation, Bu
                 new Node\Expr\Variable($varName),
                 new Node\Expr\Ternary(
                     new Node\Expr\BinaryOp\Identical(
-                        new Node\Expr\Variable($relation->getPropertyName()),
-                        $this->createNullConstant()
+                        $this->createNullConstant(),
+                        new Node\Expr\Variable($relation->getPropertyName())
                     ),
                     $this->createNullConstant(),
                     new Node\Expr\Variable('this')
                 )
             ))
         );
 
-        // if ($newUserProfile !== $user->getUserProfile()) {
```

Comment:
What prompted needing to flip all of these?

Reply:
Running php-cs-fixer